api/src/main/resources/workbench.yaml

```diff
@@ -583,88 +598,77 @@ paths:
           schema:
             $ref: '#/definitions/ErrorResponse'
 
-  /v1/clusters/{billingProjectId}/{workspaceFirecloudName}:
+  /v1/clusters/{workspaceNamespace}:
```

Comment:
This may be way too late, but I'm wondering if "cluster" is a concept that should be leaked outside the server. I guess we have enough specialized UI around them that it's fine.